app/theme/components.ts

```diff
@@ -0,0 +1,14 @@
+const components = {
```

Comment:
Should we apply it across the board? https://chakra-ui.com/docs/styled-system/customize-theme#theme-extension-withdefaultcolorscheme

Reply:
These colors are some variation of red. Do we want all our components to have a theme based on red? 
We could also use the `withDefaultColorScheme` syntax but only for specific components. I think that is cleaner than the current syntax

Reply:
Just made an update by applying it the way @humphd mentioned. Since we are using red as our primary colour, it makes sense to apply it to all components at once.